src/CabanaPD_Types.hpp

```diff
@@ -14,19 +14,27 @@
 
 namespace CabanaPD
 {
+// Fracture types.
```

Comment:
Should this be "Mechanics types" instead?

Reply:
Fixed